src/components/Fieldset.tsx

```diff
@@ -0,0 +1,87 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import "lbh-frontend/lbh/components/lbh-fieldset/_fieldset.scss";
+
+/**
+ * The proptypes for {@link FieldsetLegend}.
+ */
+export interface FieldsetLegendProps {
+  id?: string;
+  className?: string;
+  isPageHeading?: boolean;
+  children?: React.ReactNode;
+}
+
+/**
+ * The type of a valid fieldset legend component. It allows any React functional
+ * components that accept {@link FieldsetLegendProps} as proptypes.
+ */
+export type FieldsetLegendType = React.FunctionComponent<FieldsetLegendProps>;
+
+/**
+ * The legend for {@link Fieldset}.
```

Comment:
```suggestion
 * A component to set a legend for a {@link Fieldset} by passing a {@link FieldsetProps.legend}.
```

Also include a usage example either here or in `Fieldset`'s docstring, so demonstrate how to provide a legend in addition to some inputs.